src/Application.php

```diff
@@ -48,12 +46,11 @@ public function bootstrap(): void
         // Call parent to load bootstrap from files.
         parent::bootstrap();
 
-        if (PHP_SAPI !== 'cli') {
-            FactoryLocator::add(
-                'Table',
-                (new TableLocator())->allowFallbackClass(false),
-            );
-        }
+        // Use if you need fallback tables
+        //\Cake\Datasource\FactoryLocator::add(
+        //    'Table',
+        //    (new \Cake\ORM\Locator\TableLocator())->allowFallbackClass(false),
+        //);
```

Comment:
```suggestion
        FactoryLocator::add('Table', (new TableLocator())->allowFallbackClass(false));
```

This call must rename to disable fallbacks, only the `if` block needs to be removed. The default behavior is to auto use `Table` class instance if a class is not found for a model, which we *do not* want.